src/main/java/com/twilio/survey/controllers/SMSTwiMLMessageFactory.java

```diff
@@ -0,0 +1,31 @@
+package com.twilio.survey.controllers;
+
+import com.twilio.sdk.verbs.*;
+import com.twilio.survey.Server;
+import com.twilio.survey.models.Survey;
+import com.twilio.survey.util.Question;
+
+public class SMSTwiMLMessageFactory extends AbstractMessageFactory {
+    public String goodByeTwiMLMessage() throws TwiMLException {
+        TwiMLResponse twiml = new TwiMLResponse();
+        twiml.append(new Message("Your responses have been recorded. Thank you for your time!"));
+        return twiml.toXML();
+    }
+
+    public String firstTwiMLQuestion(Survey survey) throws TwiMLException {
+        TwiMLResponse response = new TwiMLResponse();
+        response.append(new Message("Thanks for taking our survey."));
+        return tempMethod(survey, response);
+    }
+
+    public String nextTwiMLQuestion(Survey survey) throws TwiMLException {
+        return tempMethod(survey, null);
+    }
+
+    private String tempMethod(Survey survey, TwiMLResponse baseResponse) throws TwiMLException {
```

Comment:
Looks like this method got lost as part of some refactor. What do you think about renaming it?